From bfff198bb1e1d09c1f3398577c076b945a62c4f7 Mon Sep 17 00:00:00 2001
From: Allen Wittenauer <aw@apache.org>
Date: Mon, 9 Feb 2015 13:58:50 -0800
Subject: [PATCH 602/642] HDFS-7546. Document, and set an accepting default for dfs.namenode.kerberos.principal.pattern (Harsh J via aw)

(cherry picked from commit 63613c79c1042ea3d7706ed6f7eccc8cf48ff6ea)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt
---
 .../src/main/resources/hdfs-default.xml            |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
index d36578d..e4cb528 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
@@ -1313,6 +1313,15 @@
 </property>
 
 <property>
+  <name>dfs.namenode.kerberos.principal.pattern</name>
+  <value>*</value>
+  <description>
+    A client-side RegEx that can be configured to control
+    allowed realms to authenticate with (useful in cross-realm env.)
+  </description>
+</property>
+
+<property>
   <name>dfs.namenode.avoid.read.stale.datanode</name>
   <value>false</value>
   <description>
-- 
1.7.0.4

